jobs/e2e-test.groovy

```diff
@@ -272,18 +272,54 @@ def analyzeResults(foldersList) {
       return;
    }
 
-   // report-merged-results.ts is a new file
-   kaGit.safePull("webapp/services/static/dev/cypress/e2e/tools");
-
-   dir ('webapp/services/static') {
-      sh("ls ./dev/cypress/e2e/tools");
-      // report-merged-results returns a non-zero rc if it detects test
-      // failures. We set the job to UNSTABLE in that case (since we don't
-      // consider smoketest failures to be blocking). But we still keep on
-      // running the rest of this script!
-      catchError(buildResult: "UNSTABLE", stageResult: "UNSTABLE",
-         message: "There were test failures!") {
-         exec(["npx", "--yes", "tsx", "./dev/cypress/e2e/tools/report-merged-results.ts", *foldersList]);
+   withTimeout('5m') {
+      // several new files in util and tools
+      kaGit.safePull("webapp/services/static/dev/cypress/e2e/tools");
+      kaGit.safePull("webapp/services/static/dev/cypress/e2e/util");
+
+      dir('webapp/services/static') {
+         // Get the SLACK_TOKEN from global credentials (env vars).
```

Comment:
Wait, what is this?

We should be getting the slackbot token from gsm, not from jenkins.  Was SLACK_TOKEN something you set up, or something that was already there?

Reply:
@jandrade this looks like something you added?  I think this is something I missed when reviewing #98, on account of that PR being a rename of a file that maybe I didn't review originally.